src/lib/ClassPreferences.js

```diff
@@ -0,0 +1,40 @@
+const VERSION = 'v1'; // In case we ever need to invalidate these
```

Comment:
I would give this an int value (without the 'v'), in case we want to define ranges.